### PR TITLE
Added a name and shame for Lloyds Bank

### PR DESCRIPTION
**Pull request by DX-MON**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -410,6 +410,17 @@ Oh and also look we got pages **NOT TRANSLATED IN FRENCH** because duh.
 
 |Izly by Crous|
 
+`Lloyds Bank <https://online.lloydsbank.co.uk/personal/logon/login.jsp>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Max 15 characters, min 8. You cannot use **ANY** sepcial characters -
```
Comment (duffn) [inline README.rst]: Minor misspelling of `special` here.

Comment (DX-MON) [reply README.rst]: Ah, well spotted! Fixed.

**Comment by duffn** [comment]
Thanks @DX-MON. This also needs an entry at the bottom with the list of images: https://github.com/dumb-password-rules/dumb-password-rules/blob/1c0f3a4378d4dd2d92e1679e6627dcf2672b38fd/README.rst#L947

**Comment by DX-MON** [comment]
Ah, I had wondered how that was working, thanks for the heads up @duffn, that's fixed now along with the merge conflict.

**Comment by duffn** [comment]
Thank you!

**Pull request merged**